cmd/gazelle/fix-update.go

```diff
@@ -195,8 +195,15 @@ func runFixUpdate(cmd command, args []string) error {
 
 	// Visit all directories in the repository.
 	var visits []visitRecord
+	var replKinds = map[config.ReplacementKind]struct{}{} // record all known ReplacementKinds
```

Comment:
This doesn't have enough information. `map_kind` directives should not be global; they should only be effective in the directory where they are set and in subdirectories.

Reply:
Yeah, good point. I'll add it to the visitRecord.